aggregator_api/Cargo.toml

```diff
@@ -10,20 +10,20 @@ version.workspace = true
 
 [dependencies]
 anyhow.workspace = true
-async-trait = "0.1"
+async-trait = { workspace = true }
 base64.workspace = true
 derivative.workspace = true
 janus_aggregator_core.workspace = true
 janus_core.workspace = true
 janus_messages.workspace = true
 opentelemetry.workspace = true
-querystring = "1.1.0"
+querystring = { workspace = true }
 rand = { workspace = true, features = ["min_const_gen"] }
-ring = "0.17.8"
+ring = { workspace = true }
 serde.workspace = true
 serde_json.workspace = true
 thiserror.workspace = true
-tracing = "0.1.40"
+tracing = { workspace = true }
```

Comment:
Ideally, rather than having two formats (`foo.workspace = true`, `foo = { workspace = true }`) we'd settle on just one. I'm fine with either.

Reply:
I agree, but I don't think `cargo autoinherit` supports choosing a style like that.

Reply:
https://github.com/mainmatter/cargo-autoinherit/issues/15